lib/main_common.pm

```diff
@@ -1202,6 +1202,7 @@ sub load_x11tests {
     }
     # first module after login or startup to check prerequisites
     loadtest "x11/desktop_runner";
+    loadtest "locale/keymap_or_locale_x11";
```

Comment:
As this relies on xterm as well could you move it after the corresponding module?

Reply:
done